components/wifi/esp8266-driver/ESP8266Interface.cpp

```diff
@@ -893,7 +893,7 @@ nsapi_error_t ESP8266Interface::set_country_code(bool track_ap, const char *coun
 
     // Firmware takes only first three characters
     strncpy(_ch_info.country_code, country_code, sizeof(_ch_info.country_code));
-    _ch_info.country_code[sizeof(_ch_info.country_code)-1] = '\0';
+    _ch_info.country_code[sizeof(_ch_info.country_code) - 1] = '\0';
```

Comment:
For future - styling changes should be separate (I got to this line because its causing merge errors when applying this patch - someone else changed this line elsewhere)

Reply:
Same for the line above, these 2 are causing conflicts when doing new release candidate

Reply:
@0xc0170 , just to make sure - you mean "separate commit", not "separate PR"?

Reply:
I was forced to change this line by the Travis astyle check, it is likely that another PR faced the same issue. 
No idea how astyle missed this when it was committed first, but it is checking the whole file that is being comitted :(
Would it help if the change was in a separate commit?

Reply:
correct separate commit at least. I've identified this, 2 PR have t he same changes - #10265

Reply:
Still having issues with this patch - changing the style for a commit that is targeting 5.13 (country code). I'll move this to 5.13 . Please send separate patch for 5.12 (targeting 5.12 branch) to be included in 5.12.2 release. We will patch manually.